### PR TITLE
[MODORDERS-784] - Logging Improvement - Configuration

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose

https://issues.folio.org/browse/MODORDERS-784
#### TODOS and Open Questions

The property file and all logging commands have been adjusted.
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by SerhiiNosko — APPROVED** [review]

**Comment by azizbekxm** [comment]
image

**Comment by azizbekxm** [comment]
image

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -13,13 +13,13 @@
   Why are you making this change? There is nothing more important
   to provide to the reviewer and to future readers than the cause
   that gave rise to this pull request. Be careful to avoid circular
-  statements like "the purpose is to update the schema." and
-  instead provide an explanation like "there is more data to be provided and stored for Purchase Orders 
+  statements like "the purpose is to update the schema." 
+  instead, provide an explanation like "there is more data to be provided and stored for Purchase Orders
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:17]: Why do we need a comma sign here?

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -31,52 +31,59 @@
...
   with your solution. Don't let all that hard work go to waste! A
   pull request is a *perfect opportunity to share the learning that
-  you did. Add links to blog posts, patterns, libraries or addons used
+  you did. Add links to blog posts, patterns, libraries, or add-ons used
   to solve this problem.
 -->
 
 ## Pre-Merge Checklist:
-Before merging this PR, please go through the following list and take appropriate actions.
+Before merging this PR, please go through the following list and take appropriate action.
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:55]: "Action" should be in the plural form

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -31,52 +31,59 @@
...
 
 - Does this PR meet or exceed the expected quality standards?
   - [ ] Code coverage on new code is 80% or greater
-  - [ ] Duplications on new code is 3% or less
+  - [ ] Duplications on new code are 3% or less
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:59]: Put it back the way it was

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -31,52 +31,59 @@
...
   - [ ] There are no major code smells or security issues
 - Does this introduce breaking changes?
-  - [ ] Were any API paths or methods changed, added or removed?
+  - [ ] Were any API paths or methods changed, added, or removed?
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:62]: Remove comma here

Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md]: ok, I have done

File: PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -31,52 +31,59 @@
...
 
 ## Learning
 <!-- OPTIONAL
-  Help out not only your reviewer, but also your fellow developer!
+  Help out not only your reviewer but also your fellow developer!
   Sometimes there are key pieces of information that you used to come up
   with your solution. Don't let all that hard work go to waste! A
   pull request is a *perfect opportunity to share the learning that
-  you did. Add links to blog posts, patterns, libraries or addons used
+  you did. Add links to blog posts, patterns, libraries, or add-ons used
```
[the remark this answers is not in the document]
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md]: ok, I have done

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:17]: recommended by Grammarly

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**